src/web/components/Avatar.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+import { pillarPalette } from '@frontend/lib/pillars';
+
+const contributorImage = css`
+    border-radius: 100%;
+    object-fit: cover;
```

Comment:
The fallback for this for users of IE11 and earlier Edge is (presumably) pretty graceful

![2019-11-12 17 12 27](https://user-images.githubusercontent.com/1336821/68693753-bbe3d200-056f-11ea-9eec-eab1b43cab29.gif)